docs/en/02_Developer_Guides/14_Files/07_File_Usage.md

```diff
@@ -0,0 +1,84 @@
+---
+title: File usage
+summary: See file usage and customising the file "Used on" table
+icon: compress-arrows-alt
+---
+
+# File Usage
+
+CMS users can view where a file is used by accessing the "Used On" tab in the file editor. This feature allows them to identify what DataObjects depend on the File.
```

Comment:
```suggestion
CMS users can view where a file is used by accessing the Used on tab in the Files section. This feature allows them to identify what DataObjects depend on the file.
```

Reply:
These suggestions are more in line with the writing structure here: https://userhelp.silverstripe.org/en/4/creating_pages_and_content/creating_and_editing_content/images_and_documents/#file-usage